miden/src/cli/bundle.rs

```diff
@@ -29,29 +36,42 @@ impl BundleCmd {
         println!("Build library");
         println!("============================================================");
 
-        let namespace = match &self.namespace {
-            Some(namespace) => namespace.to_string(),
-            None => self
-                .dir
-                .file_name()
-                .expect("dir must be a folder")
-                .to_string_lossy()
-                .into_owned(),
-        };
-
-        let assembler = Assembler::default().with_debug_mode(true);
-        let library_namespace =
-            namespace.parse::<LibraryNamespace>().expect("invalid base namespace");
-        let library = Library::from_dir(&self.dir, library_namespace, assembler)?;
+        let assembler = Assembler::default().with_debug_mode(self.debug);
 
         // write the masl output
         let output_file = self
             .dir
             .join(self.namespace.as_deref().unwrap_or("out"))
             .with_extension(Library::LIBRARY_EXTENSION);
```

Comment:
I think it would be more conventional to add a `-o`/`--output-file`. Now, the file name is either the namespace, or `out` (with no way of configuring the file name without changing the namespace). So the default name should be `out.masl` (as is currently), and only changed by a new `-o` option

Reply:
done